### PR TITLE
PMM-10874-Register-k8s-Cluster-Move-to-the-separate-page: add k8s reg…

### PR DESCRIPTION
**Pull request by solovevayaroslavna**
https://jira.percona.com/browse/PMM-10874

[Запись встречи 14.11.2022 14-25-09 - запись.webm](https://user-images.githubusercontent.com/90199600/201636831-d7bcf5b5-c787-4903-a1c9-9e33d8208925.webm)

### CODE REVIEW

File: public/app/percona/shared/core/reducers/k8sCluster/k8sCluster.types.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export interface PerconaK8SClusterState {
+  result: 'ok' | 'error' | undefined;
```
Comment (fabio-silva) [inline public/app/percona/shared/core/reducers/k8sCluster/k8sCluster.types.ts:2]:
```suggestion
  result?: 'ok' | 'error';
```

File: public/app/percona/shared/core/reducers/k8sCluster/k8sCluster.types.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+  loading: boolean | undefined;
```
Comment (fabio-silva) [inline public/app/percona/shared/core/reducers/k8sCluster/k8sCluster.types.ts:3]:
```suggestion
  loading?: boolean;
```

**Review by fabio-silva — CHANGES_REQUESTED** [review]

File: public/app/percona/dbaas/components/Kubernetes/EditK8sClusterPage/EditK8sClusterPage.styles.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        justify-content: space-between;
+      }
+    }
+    &:not(:last-child) {
+      margin-bottom: 0px;
+    }
+  `,
+
+  awsField: css`
+    max-width: 793px;
```
[the remark this answers is not in the document]
Comment (solovevayaroslavna) [reply public/app/percona/dbaas/components/Kubernetes/EditK8sClusterPage/EditK8sClusterPage.styles.ts:57]: I will ask Nuna about this number, thanks!

File: public/app/percona/dbaas/components/Kubernetes/EditK8sClusterPage/EditK8sClusterPage.styles.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+  pasteButton: css`
+    background-color: red;
+  `,
+  pageToolbarWrapper: css`
+    display: flex;
+    align-items: flex-start;
+    padding-right: ${spacing.lg};
+  `,
+
```
[the remark this answers is not in the document]
Comment (solovevayaroslavna) [reply public/app/percona/dbaas/components/Kubernetes/EditK8sClusterPage/EditK8sClusterPage.styles.ts:23]: fixed

File: public/app/percona/dbaas/components/Kubernetes/EditK8sClusterPage/EditK8sClusterPage.styles.ts [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+export const getStyles = ({ spacing }: GrafanaTheme) => ({
+  checkbox: css`
+    span {
+      top: 0;
+    }
+  `,
+  urlWarningWrapper: css`
+    margin-bottom: ${spacing.md};
+  `,
+
```
[the remark this answers is not in the document]
Comment (solovevayaroslavna) [reply public/app/percona/dbaas/components/Kubernetes/EditK8sClusterPage/EditK8sClusterPage.styles.ts:14]: fixed

File: public/app/percona/dbaas/components/Kubernetes/EditK8sClusterPage/EditK8sClusterPage.styles.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (solovevayaroslavna) [reply public/app/percona/dbaas/components/Kubernetes/EditK8sClusterPage/EditK8sClusterPage.styles.ts:57]: fixed to 800px

File: public/app/percona/backup/components/AddBackupPage/AddBackupPage.tsx [hunk 6]
```diff
@@ -110,6 +112,16 @@ const AddBackupPage: FC<GrafanaRouteComponentProps<{ type: string; id: string }>
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const onDemandChange = useCallback(() => {
+    setQueryParams({ scheduled: null });
+    setModalTitle(Messages.getModalTitle(false, editing));
+  }, [editing, setQueryParams]);
+
+  const onSheduledChange = useCallback(() => {
```
Comment (fabio-silva) [inline public/app/percona/backup/components/AddBackupPage/AddBackupPage.tsx:120]: Should be "scheduled"

**Review by fabio-silva — CHANGES_REQUESTED** [review]

File: public/app/percona/backup/components/AddBackupPage/AddBackupPage.tsx (hunk 6 above)
Comment (solovevayaroslavna) [reply public/app/percona/backup/components/AddBackupPage/AddBackupPage.tsx:120]: Are you sure? 
image
I checked how it was in previous version and there was 2 times "editing"

Comment (fabio-silva) [reply public/app/percona/backup/components/AddBackupPage/AddBackupPage.tsx:120]: I mean there's a typo: "onScheduledChange" and not "onSheduledChange" 😅

Comment (solovevayaroslavna) [reply public/app/percona/backup/components/AddBackupPage/AddBackupPage.tsx:120]: image

Comment (solovevayaroslavna) [reply public/app/percona/backup/components/AddBackupPage/AddBackupPage.tsx:120]: or) yes) sorry! I see now

Comment (solovevayaroslavna) [reply public/app/percona/backup/components/AddBackupPage/AddBackupPage.tsx]: fixed

**Review by matejkubinec — DISMISSED** [review]

**Review by fabio-silva — DISMISSED** [review]

**Review by matejkubinec — DISMISSED** [review]

**Review by matejkubinec — APPROVED** [review]

**Pull request merged**